commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemServiceImpl.java

```diff
@@ -147,6 +147,34 @@ public CommerceInventoryWarehouseItem getCommerceInventoryWarehouseItem(
 				commerceInventoryWarehouseId, start, end);
 	}
 
+	@Override
+	public List<CommerceInventoryWarehouseItem>
+			getCommerceInventoryWarehouseItemsByCompanyId(
+				long companyId, int start, int end)
+		throws PortalException {
+
+		PortalPermissionUtil.check(
+			getPermissionChecker(),
+			CommerceInventoryActionKeys.MANAGE_INVENTORY);
+
+		return commerceInventoryWarehouseItemLocalService.
+			getCommerceInventoryWarehouseItemsByCompanyId(
+				companyId, start, end);
+	}
+
+	@Override
+	public int getCommerceInventoryWarehouseItemsByCompanyIdCount(
```

Comment:
Here I guess we can't take out the `ByCompanyId` :(. But the name should be `getCommerceInventoryWarehouseItemsCountByCompanyId` instead of `getCommerceInventoryWarehouseItemsByCompanyIdCount`.

Reply:
Ok, I'll rename that